### PR TITLE
feat: created command create with subcommand from variable

### PR DESCRIPTION
**Pull request by maxwelbm**
- add sub command create for variables command tree;
- unit tests;

### CODE REVIEW

**Review by pauloSF0 — APPROVED** [review]

File: pkg/cmd/variables/create/create.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	fields := &Fields{}
+
+	cmd := &cobra.Command{
+		Use:           msg.CreateUsage,
+		Short:         msg.CreateShortDescription,
+		Long:          msg.CreateLongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+        $ azioncli variables create --key "key" --value "value" --secret false
```
Comment (PatrickMenoti) [inline pkg/cmd/variables/create/create.go:35]: I think we can put an actual good example, that theoretically could be copy+pasted, such as:
`azioncli variables create --key "Content-Type" --value "string" --secret false`

File: messages/variables/errors.go [hunk 2]
```diff
@@ -8,4 +8,6 @@ var (
 	ErrorGetVariables                    = errors.New("Failed to describe the variables: %s. Check your settings and try again. If the error persists, contact Azion support.")
 	ErrorFailToDeleteVariable            = errors.New("Failed to delete the Variable: %s. Check your settings and try again. If the error persists, contact Azion support")
 	ErrorMissingVariableIdArgumentDelete = errors.New("A mandatory flag is missing. You must provide a variable_id as an argument. Run the command 'azioncli variables <subcommand> --help' to display more information and try again")
+	ErrorMandatoryCreateFlags            = errors.New("One or more required flags are missing. You must provide --key, --name, and --secret flags when the --in flag is not provided. Run the command 'azioncli edge_applications create --help' to display more information and try again")
```
Comment (PatrickMenoti) [inline messages/variables/errors.go:11]: change `edge_applications` to `variables`

File: messages/variables/messages.go [hunk 3]
```diff
@@ -32,4 +32,15 @@ var (
...
 	DeleteLongDescription  = "Delete a Variable using UUID"
+
+	// [ create ]
+	CreateUsage            = "create [flags]"
+	CreateShortDescription = "Creates a new variable"
+	CreateLongDescription  = "Creates a new variable"
+	CreateFlagKey          = "Informs  Key"
+	CreateFlagValue        = "Informs  value"
+	CreateFlagSecret       = "Informs  Secret"
+	CreateFlagIn           = "Path to a JSON file containing the attributes of the origin that will be created; you can use - for reading from stdin"
```
Comment (PatrickMenoti) [inline messages/variables/messages.go:43]: Path to a JSON file containing the attributes of the `variable` that will be created; you can use - for reading from stdin

File: messages/variables/messages.go [hunk 4]
```diff
@@ -32,4 +32,15 @@ var (
...
+	CreateOutputSuccess    = "Created origin with ID %s\n"
```
Comment (PatrickMenoti) [inline messages/variables/messages.go:44]: change `origin` to `variable`

**Review by PatrickMenoti — CHANGES_REQUESTED** [review]

**Review by PatrickMenoti — DISMISSED** [review]

**Review by PatrickMenoti — APPROVED** [review]

**Review by peterpaulo-azion — APPROVED** [review]

**Pull request merged**